presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlIntegrationSmokeTest.java

```diff
@@ -169,6 +169,40 @@ public void testInsertWithFailureDoesntLeaveBehindOrphanedTable()
         }
     }
 
+    @Test
+    public void testDecimalPredicatePushdown()
+            throws Exception
+    {
+        try (AutoCloseable ignoreTable = withTable("tpch.test_decimal_pushdown",
+                "(short_decimal decimal(9, 3), long_decimal decimal(30, 10))")) {
+            execute("INSERT INTO tpch.test_decimal_pushdown VALUES (123.321, 123456789.987654321)");
+
+            assertQuery("SELECT * FROM tpch.test_decimal_pushdown WHERE short_decimal <= 123.321",
+                    "VALUES (123.321, 123456789.987654321)");
+            assertQuery("SELECT * FROM tpch.test_decimal_pushdown WHERE long_decimal <= 123456789.987654321",
```

Comment:
there should be a test with `=`, not only `<=`.
And one with `<=`  where value is bigger than the searched one.